apps/cowswap-frontend/src/legacy/state/orders/utils.ts

```diff
@@ -187,7 +191,42 @@ export function getOrderMarketPrice(order: Order, quotedAmount: string, feeAmoun
  * 5% to level out the fee amount changes
  */
 const EXECUTION_PRICE_FEE_COEFFICIENT = new Percent(5, 100)
+const FEE_AMOUNT_MULTIPLIER = 50
```

Comment:
Sorry, I don't understand, why 50?

Reply:
That's what I showed during the call today.

We use a fixed amount for partial fills to be able to have a stable price for varying order sizes.
So a order with 10 eth and one with 1000 eth would have the same estimated execution price given that they can be partially fillable.

Anxo's suggestion is to use 50, but as we could see today that might be too little.
The new proposal is to use 1000, so the fee in relation to the sell amount corresponds to 0.1%

Reply:
I see, thank you!